examples/lessons/15/index.js

```diff
@@ -1,8 +1,10 @@
-function webGLStart() {
+var webGLStart = function() {
   var $id = function(d) { return document.getElementById(d); };
-  
+
+  var pgl = PhiloGL;
+
   //Create moon
-  var earth = new PhiloGL.O3D.Sphere({
+  var earth = new pgl.O3D.Sphere({
```

Comment:
I think we should drop the intermediary O3D namespace, and just export the Models directly. 

Can't see that it adds anything.